sdk/identity/Azure.Identity/src/ProcessRunner.cs

```diff
@@ -25,6 +25,7 @@ internal sealed class ProcessRunner : IDisposable
 
         public ProcessRunner(IProcess process, TimeSpan timeout, CancellationToken cancellationToken)
         {
+            AzureIdentityEventSource.Singleton.ProcessRunnerInformational($"Running process `{process.StartInfo.FileName}' with arguments {string.Join(", ", process.StartInfo.Arguments)}");
```

Comment:
I believe the arguments will contains a resource string in most cases, as well as tenant id if it was specified by the user. We've talked before about whether these are considered PII in the past, but I don't recall the outcome. @g2vinay are we ok to log resource strings and tenant ids or should they be redacted?

Reply:
I believe last we decided these were not PII.
In Java scopes are logged in other credentials when a token is retrieved.

Reply:
If I recall correctly, tenantID was not PII but resource strings do get close to being PII depending on the resource name. I'll try to find my notes about this.

Reply:
Just checked tenantID is OII (organization identifiable data) so not PII, but I think resources (and therefore their resource strings?) do fall under customer content. I can share the link I have for the data governance site on our teams channel.

Reply:
Maybe, just to be extra cautious, this change should be coupled with the upcoming setting to log PII information (previously only relevant for MSAL logging).